DEVELOPER_GUIDE.md

```diff
@@ -53,6 +50,15 @@ opensearch.password: "admin" # Default password on the docker image
 opensearch.ssl.verificationMode: none
 ```
 
+### Run Opensearch
+
+You need to have an OpenSearch server up and running to be able to run OpenSearch
+Dashboards. In a separate terminal you can run the latest nightly build using
+
+```bash
+$ yarn opensearch snapshot 
```

Comment:
I believe this will run into the same issues that this issue ran into: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1263.

Should we make a reference about this or warning?

Reply:
Nice catch, thanks! This is exactly how i ran into it. I'll add a warning here to call that out.